test/__setup__/console.ts

```diff
@@ -1,22 +0,0 @@
-/* eslint-disable jest/no-standalone-expect */
```

Comment:
@lukashass Is there no alternative for vitest to capture `console.error` calls?

Reply:
@anbraten I added an alternative in [f5e1675](https://github.com/geprog/use-feathers/pull/74/commits/f5e1675004893258eeed53c869ffdb47cd61881d).